source/EscortDisplay.cpp

```diff
@@ -223,7 +199,11 @@ bool EscortDisplay::Icon::operator<(const Icon &other) const
 
 int EscortDisplay::Icon::Height() const
 {
-	return 30 + 15 * !system.empty();
+	const Interface *element = GameData::Interfaces().Get("escort element");
```

Comment:
This interface is used a lot in multiple functions, you might as well store it as a member object.